From 6c95dbcdcc4bc501c7c1921159b1a3846a463ab9 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sat, 5 Nov 2022 02:00:16 -0500
Subject: [PATCH 1/1] remove /etc/issue from tmpfiles

This tries to copy whatever is at /usr/share/factory/etc/issue, but we don't ship anything there so systemd prints an error.
---
 tmpfiles.d/etc.conf.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tmpfiles.d/etc.conf.in b/tmpfiles.d/etc.conf.in
index d856625320..dda61a0d56 100644
--- a/tmpfiles.d/etc.conf.in
+++ b/tmpfiles.d/etc.conf.in
@@ -12,4 +12,3 @@ L+ /etc/mtab - - - - ../proc/self/mounts
 {% if HAVE_SMACK_RUN_LABEL %}
 t /etc/mtab - - - - security.SMACK64=_
 {% endif %}
-C! /etc/issue - - - -
-- 
2.38.1

